### ISSUE TITLE
[IceHive] code assumes input file has certain format (Trac #1791)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1783]
if the input file does not start with a `#`  it returns garbage. can probably fixed with an `else{log_fatal()}` at line 448

found by static analysis http://software.icecube.wisc.edu/static_analysis/2016-07-26-030212-26135-1/report-14f501.html#EndPath

Migrated from https://code.icecube.wisc.edu/ticket/1791
```json
{
    "status": "closed", 
    "changetime": "2016-08-09T12:37:06", 
    "description": "if the input file does not start with a `#`  it returns garbage. can probably fixed with an `else{log_fatal()}` at line 448\n\nfound by static analysis http://software.icecube.wisc.edu/static_analysis/2016-07-26-030212-26135-1/report-14f501.html#EndPath", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1470746226054878", 
    "component": "combo reconstruction", 
    "summary": "[IceHive] code assumes input file has certain format", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-07-27T07:38:50", 
    "milestone": "Long-Term Future", 
    "owner": "mzoll", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160801T17:28:05: olivas changed component from "cmake" to "icerec"

**Comment by icecube-trac** [comment]
Trac update at 20160809T12:37:06: 
* mzoll commented:

> In changeset 149145:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149145"
> explicitly assume certain input file format and terminate if error; fixes #1783
> ```


* mzoll changed status from "new" to "closed"
* mzoll changed resolution from "" to "fixed"
